### PR TITLE
Added prefix and suffix props to EmailInput

### LINKED ISSUE #948: Add prefix and suffix options to EmailInput
Let's add the option to add prefix and suffix props to `EmailInput` just like we have in the `Input` component.

image

### PR DESCRIPTION
**Pull request by mohit-s96**
closes #948

### CODE REVIEW

**Comment by mohit-s96** [comment]
@ajmaln _a. Please review.

**Comment by ajmaln** [comment]
@mohit-s96 _a let's make the prefix and suffix controllable in storybook like in `Button` story.
image

**Comment by mohit-s96** [comment]
New controls for `suffix` and `prefix`. Limited the options to a few icons and subdomains.

suffix-prefix-controls-storybook

**Comment by mohit-s96** [comment]
@ajmaln _a. Please review the changes.

File: stories/constants.js [hunk 1]
```diff
@@ -8,7 +19,26 @@ export const icons = {
   Down,
   Left,
   Right,
-  Download
+  Download,
+};
+
+const makePrefix = (Icon) => <div className="pl-2">{<Icon />}</div>;
```
Comment (ajmaln) [inline stories/constants.js:25]: I think the padding should be applied on the `EmailInput` component itself like we have in the `Input` component.

**Review by ajmaln — COMMENTED** [review]
@mohit-s96 _a Please check the suggestion

**Comment by mohit-s96** [comment]
@ajmaln _a. Please review the changes.

File: stories/Components/EmailInput.stories.jsx [hunk 2]
```diff
@@ -74,6 +86,34 @@ export const Counter = () => {
...
+export const WithPrefixAndSuffix = (args) => {
+  const [emails, setEmails] = useState(args.value);
+  
+  return (
+    <EmailInput
+      {...args}
+      prefix={
+        <div className="pl-2">
+          <Search />
+        </div>
```
Comment (ajmaln) [inline stories/Components/EmailInput.stories.jsx:98]:
```suggestion
          <Search />
```

File: lib/components/EmailInput.js [hunk 3]
```diff
@@ -21,16 +21,31 @@ import { hyphenize } from "../common";
...
-  <components.Control {...props}>
-    <div className="neeto-ui-overflow-hidden">{children}</div>
-  </components.Control>
-);
+const CustomControl = ({ children, ...props }) => {
+  const { prefix } = props.selectProps;
+
+  return (
+    <components.Control {...props}>
+      {prefix && <div className="pl-2">{prefix}</div>}
```
Comment (ajmaln) [inline lib/components/EmailInput.js:29]: Since we have removed tailwindcss from neetoUI, please remove the `pl-2` and update it with a custom class

**Comment by mohit-s96** [comment]
@ajmaln _a. Please review the changes.

**Review by ajmaln — APPROVED** [review]

**Pull request merged**